### PR TITLE
update cypress readme

### LINKED ISSUE #1228: Cypress Readme
**Is your feature request related to a problem? Please describe.**
Currently, there isn't an introduction to writing Cypress tests in our application. 
We should add a Readme to the Cypress directory - to explain the directory structure and best practice in adding integration tests and helper utilities. This should be accessible to developers and testers. 

**Describe the solution you'd like**
To write a `README.md` file in the [`cypress` directory](https://github.com/bbc/simorgh/tree/latest/cypress) giving a high-level overview, for any testers or developers to be able to understand how best to contribute to the integration tests. 

**Describe alternatives you've considered**
N/A

**Testing notes**
At the 'code review' stage, this should also be reviewed by a tester in a different team and feedback should be incorporated into the PR. 

Dev insight: Same as above. 

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by thekp**
Resolves #1228

**Overall change:** _Update Cypress readme._

**Code changes:**

- _Added headings to organise the "hints" sections_

**Requires a tester to review when in code review:**
Please tell me any more information to add, if you see something missing/wrong 😄 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by thekp** [comment]
i will squash the commits when i merge

File: cypress/README.md [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+
+Use the scripts from Simorgh's [package.json](https://github.com/bbc/simorgh/blob/latest/package.json#L39#L40) to run Cypress test locally on LIVE/TEST environment.
+
+## Best Practises
+
+[Cypress best practices](https://docs.cypress.io/guides/references/best-practices.html)
+[Cypress defaults commands](https://docs.cypress.io/api/api/table-of-contents.html)
 
-- Use the default cypress commands e.g. cy.get().
+- Use the custom commands in `/support/command.js` directory.
```
Comment (jamesbhobbs) [inline cypress/README.md:34]: Should be that we use the default commands and the custom commands, we don't want a custom command created that is superficially different to the deafult. e.g. the getElement helper currently in /support.

File: cypress/README.md [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
-- Write as many tests as you please, but don't make them atomic. i.e. use .and() instead of a second test.
-- The runtime of tests matters so make each as fast as you can, avoid repeating expensive things like visit()s and request()s as much as possible.
-- Although the runtime of each test matters and making each test not too small is good practice always tend towards full coverage rather than limiting the scope of testing.
-- We could write many more best practices but cypress already did: [Cypress.io Best Practices](https://docs.cypress.io/guides/references/best-practices.html)
-- Use the scripts from the package.json to run Cypress test locally on LIVE/TEST: https://github.com/bbc/simorgh/blob/latest/package.json#L39#L40
+- Write as many tests as you please, but don't make them atomic. i.e. use .and() instead of a second test. -- The runtime of tests matters so make each as fast as you can, avoid repeating expensive things like visit()s and request()s as much as possible. -- Although the runtime of each test matters and making each test not too small is good practice always tend towards full coverage rather than limiting the scope of testing.
```
Comment (jamesbhobbs) [inline cypress/README.md]: You've broken the md here

File: cypress/README.md [hunk 3]
```diff
@@ -1,25 +1,46 @@
...
+
+<!-- prettier-ignore -->
+    .
+    ├── fixtures                    # Cypress pre-defined
+    ├── integration                 # Cypress tests
+    ├── plugins                     # Internal behavior of Cypress
+    └── support                     # Helper utilities for tests
+        └── config*
+
+\*Config exists in one place. Config for running tests is in /support/config/ and config for expected results should either be hardcoded in a test or imported from application config (for example service translations).
```
Comment (j-pendlebury) [inline cypress/README.md]:
```suggestion
\*Config exists in one place. Config for running tests is in `/support/config/` and config for expected results should either be hardcoded in a test or imported from application config (for example service translations).
```

File: cypress/README.md [hunk 4]
```diff
@@ -1,25 +1,46 @@
 # Cypress tests and our approach to E2E testing
 
+[Introduction to Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Cypress-Is-Simple)
+
 This folder contains all the simorgh application end-to-end (E2E) tests and exclusively uses cypress as a testing framework. At present we run all E2E tests on all builds and deploys to all environments including live.
```
Comment (j-pendlebury) [inline cypress/README.md]:
```suggestion
This folder contains all the Simorgh application end-to-end (E2E) tests and exclusively uses cypress as a testing framework. At present we run all E2E tests on all builds and deploys to all environments including live.
```

File: cypress/README.md [hunk 5]
```diff
@@ -1,25 +1,46 @@
...
+## Best Practises
+
+[Cypress best practices](https://docs.cypress.io/guides/references/best-practices.html)
 
-- Use the default cypress commands e.g. cy.get().
+- Use the default cypress commands e.g. cy.get().[More defaults commands](https://docs.cypress.io/api/api/table-of-contents.html)
+- Use the custom commands in `/support/command.js` directory.
 - Code reusability is good but ease of understanding is better.
-- Use custom cypress commands, see `/support/commands.js`.
 - If you're using a let, var or const in any test/helper/command in this repo, you're probably doing it unecessarily, use cypress aliasing or chain requests.
```
Comment (j-pendlebury) [inline cypress/README.md]:
```suggestion
- If you're using a let, var or const in any test/helper/command in this repo, you're probably doing it unnecessarily, use cypress aliasing or chain requests.
```

**Review by j-pendlebury — CHANGES_REQUESTED** [review]
just a couple of typos 🙈

**Review by j-pendlebury — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
No testing needed

**Pull request merged**